lib/OpenQA/Schema/Result/GruDependencies.pm

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2014 SUSE Linux Products GmbH
```

Comment:
heh, I actually was genuinely not sure what to do about that :) i guess add a Red Hat line and bump both to 2015?